extensions/positron-assistant/package.json

```
@@ -358,6 +358,39 @@
           "positron-assistant"
         ]
       },
+      {
+        "name": "getTableSummary",
+        "displayName": "Get Table Summary",
+        "modelDescription": "Get structured information about table variables in the current session.",
+        "inputSchema": {
+          "type": "object",
+          "properties": {
+            "sessionIdentifier": {
+              "type": "string",
+              "description": "The identifier of the session that contains the tables."
+            },
+            "accessKeys": {
+              "type": "array",
+              "description": "An array of table variables to summarize.",
+              "items": {
+                "type": "array",
+                "description": "A list of access keys that identify a variable by specifying its path.",
+                "items": {
+                  "type": "string",
+                  "description": "An access key that uniquely identifies a variable among its siblings."
+                }
+              }
+            }
+          },
+          "required": [
+            "sessionIdentifier",
+            "accessKeys"
+          ]
+        },
+        "tags": [
+          "positron-assistant"
+        ]
```

Sorry, one more thing before merging! Since https://github.com/posit-dev/positron/pull/8495 has gone in, we'll also need to add this tag:
```suggestion
        "tags": [
          "positron-assistant",
          "requires-session"
        ]
```